presto-main/src/main/java/com/facebook/presto/operator/scalar/VarbinaryFunctions.java

```diff
@@ -146,6 +146,26 @@ public static Slice toBigEndian64(@SqlType(StandardTypes.BIGINT) long value)
         return slice;
     }
 
+    @Description("encode value as an IEEE 754 floating-point 'single format' big endian varbinary")
```

Comment:
I understand this comes from the Java doc:
```
Returns a representation of the specified floating-point value
according to the IEEE 754 floating-point "double format" bit layout.
```

However, my understanding is that `IEEE 754 floating-point "double format" bit layout` is a whole thing. And make it to be `IEEE 754 floating-point 'single format' big endian varbinary`.

What about the following (tentative, we will see how others think later :) ):
```
encode value as a big endian varbinary according to IEEE 754 single-precision floating-point format 
```

`single-precision floating-point format` comes from https://en.wikipedia.org/wiki/Single-precision_floating-point_format